I/ImageIO/Compat.toml

```diff
@@ -1,8 +1,11 @@
-[0]
+["0.0.1-0.1"]
+julia = "1"
+
+["0.1-0"]
 FileIO = "1.2.0-1"
 julia = "1.3.0-1"
 
-["0-0.1.0"]
+["0.1.0-0.1.1"]
```

Comment:
I guess the CI is complaining about this change...

I could absorb this `["0.1.0-0.1.1"]` and `["0.1.1-0.1"]` to

```julia
["0.1"]
PNGFiles = "0.1.2-0.1"
```

It should also work, but that's not a related change to this PR.

It would be helpful if someone could point me what the correct way to write this.

Reply:
@DilumAluthge May I get some suggestions for this? Or redirect me to someplace where the registry compat.toml syntax is documented?

Reply:
Hmmm. Probably the easiest approach here is for you to use LocalRegistry.jl to generate the commit. Then push the commit up to your fork, and make the PR. LocalRegistry.jl will make sure that all the `Compat.toml` syntax and compression is done correctly.

I think the LocalRegistry.jl docs are pretty good. If you run into trouble, we can probably get help from @GunnarFarneback.

Reply:
If you use LocalRegistry.jl, you won't have to worry about editing any of these files by hand, which I think is the best approach.

Reply:
Something like this, although you might want to target your fork of General directly instead of `~/.julia/registries/General`.
```
gunnar@debian19:~/.julia/registries/General$ cd ~/.julia/dev/ImageIO/
gunnar@debian19:~/.julia/dev/ImageIO$ git checkout release-0.0.1
Already on 'release-0.0.1'
Your branch is up to date with 'origin/release-0.0.1'.
gunnar@debian19:~/.julia/dev/ImageIO$ julia --project
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.5.0 (2020-08-01)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> using LocalRegistry

julia> register(commit = false)
┌ Info: Registering package
│   package_path = "/home/gunnar/.julia/dev/ImageIO"
│   registry_path = "/home/gunnar/.julia/registries/General"
│   package_repo = ""
│   uuid = UUID("82e4d734-157c-48bb-816b-45c225c6df19")
│   version = v"0.0.1"
│   tree_hash = "08e25c3447f8147f823d488e1e53d2f385d4fddd"
└   subdir = ""

julia> 
gunnar@debian19:~/.julia/dev/ImageIO$ cd ~/.julia/registries/General/
gunnar@debian19:~/.julia/registries/General$ git diff
diff --git a/I/ImageIO/Compat.toml b/I/ImageIO/Compat.toml
index 00aae5973e..0cd5a11cc7 100644
--- a/I/ImageIO/Compat.toml
+++ b/I/ImageIO/Compat.toml
@@ -1,8 +1,11 @@
-[0]
+["0.0"]
+julia = "1"
+
+["0.1-0"]
 FileIO = "1.2.0-1"
 julia = "1.3.0-1"
 
-["0-0.1.0"]
+["0.1.0"]
 PNGFiles = "0.1"
 
 ["0.1.1-0.1"]
diff --git a/I/ImageIO/Deps.toml b/I/ImageIO/Deps.toml
index de5e4c49c9..2848730896 100644
--- a/I/ImageIO/Deps.toml
+++ b/I/ImageIO/Deps.toml
@@ -1,4 +1,4 @@
-[0]
+["0.1-0"]
 FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
 PNGFiles = "f57f5aa1-a3ce-4bc8-8ab9-96f992907883"
 
diff --git a/I/ImageIO/Versions.toml b/I/ImageIO/Versions.toml
index f8d2a2080f..bb2b4d1b6a 100644
--- a/I/ImageIO/Versions.toml
+++ b/I/ImageIO/Versions.toml
@@ -1,3 +1,6 @@
+["0.0.1"]
+git-tree-sha1 = "08e25c3447f8147f823d488e1e53d2f385d4fddd"
+
 ["0.1.0"]
 git-tree-sha1 = "ce431b478f91b7631ffd2d3deff4973417485239"
 
```

Reply:
Thank you for the suggestions!